tajo-docs/src/main/sphinx/hbase_integration.rst

```diff
@@ -94,7 +99,8 @@ In order to create a new HBase table which is to be managed by Tajo, use the USI
   CREATE EXTERNAL TABLE blog (rowkey text, author text, register_date text, title text)
   USING hbase WITH (
     'table'='blog'
-    , 'columns'=':key,info:author,info:date,content:title');
+    , 'columns'=':key,info:author,info:date,content:title')
+  LOCATION 'hbase:zk://localhost:2181/';
```

Comment:
We need to specify general environment. So, you need to use ``<hostname>`` instead of ``localhost`` and ``<port>`` instead of ``2181``.